### ISSUE TITLE
Fails when <Bundler CSS start> contains links to the cdn

### ISSUE THREAD

**Issue by AntonDnepr** [ancoopa/static-pages-bundler#2]
I used sudo to run command, because of isse #1 and here is the error:

```
Error: ENOENT: no such file or directory, open 'https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css'

```

Part of the index.html is next

```
<!-- Bootstrap CSS -->
		<!-- Bundler CSS start -->
		<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css" integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous">
		<link type="text/css" rel="stylesheet" href="./css/main.css">
		<!-- Fonts -->
		<link href="https://fonts.googleapis.com/css?family=Montserrat:400,500,600,700,800,900" rel="stylesheet">
		<link rel="stylesheet" type="text/css" href="./fullPage/jquery.fullPage.min.css" />
		<!-- Bundler CSS end -->
```

I assume this should be mentioned on the docs somehow.

**Comment by ancoopa** [comment]
Hello Anton,
Thank you for the report.

The feature is not supported yet. For now `static-pages-bundler` works only with files that are located on your computer. But I think working with remote files may be the next feature.

For now, I'll add information about it to the documentation. And as soon as it will be possible, I will implement the feature.

Thank you for your help in improving the tool.

**Comment by ancoopa** [comment]
Implemented in v1.0.25
